### ISSUE TITLE
Add example/documentation for using IK solution with MocoStateTrackingGoal

### ISSUE THREAD

**Issue by chrisdembia** [opensim-org/opensim-moco#607]
Main issues are changing column labels and converting to radians.

**Comment by chrisdembia** [comment]
This was a repeated issue during the March 2020 workshop.

**Comment by carmichaelong** [comment]
Having some feedback/output on what's being done behind the scenes could also help.

For instance, the radians issue was found because of the `tracke_states.sto` file. While the input had `inDegrees=yes`, the output file had `inDegrees=no` even though all the values were the same.

**Comment by chrisdembia** [comment]
I'm not sure what is meant by the output file. Was that the MocoSolution STO file?

**Comment by carmichaelong** [comment]
Oops, output file = `tracked_states.sto` file. That's a typo.

**Comment by chrisdembia** [comment]
Oh! That's generated by MocoTrack during preprocessing. I forgot. That's a useful clue. This is a high-priority issue.

I do not understand where the issue is, given that here is where `process()` is invoked:

https://github.com/opensim-org/opensim-moco/blob/c8335c878548b137f9eeb159bdc3c647d1a96060/Moco/Moco/MocoTrack.cpp#L150-L151

and this is where the angle conversion happens in `process()`:

https://github.com/opensim-org/opensim-moco/blob/c8335c878548b137f9eeb159bdc3c647d1a96060/Moco/Moco/Common/TableProcessor.h#L103-L106

**Comment by chrisdembia** [comment]
@psbiomech have you used IK solutions as the state tracking reference in Moco?

**Comment by nickbianco** [comment]
@carmichaelong it might be helpful to have a set of files that reproduce the problem. It probably warrants it's own GitHub issue; if you have some time to reproduce and create the issue that would be helpful.

**Comment by carmichaelong** [comment]
Definitely. Will plan on trying to get a minimal example for C++ this week.
